.github/workflows/zips.yml

```diff
@@ -58,6 +58,12 @@ on:
       - synchronize
       - ready_for_review
   workflow_dispatch:
+    inputs:
```

Comment:
How about we also add a `dry_run` option here for testing?

This would essentially pass through to the respective input on the action.
https://github.com/10up/action-wordpress-plugin-deploy/blob/977dc0cce7451fdd3ce4830920f75118dfc57002/action.yml#L11-L12

Reply:
Yep, good idea. Added.